DBMS/Assignment1/library.sql

```diff
@@ -0,0 +1,128 @@
+/* created database*/
+CREATE DATABASE Library_Management;
+USE Library_Management;
+
+/* created table named members*/
+CREATE TABLE members(
```

Comment:
ACTION: Table Name should be "member". Tables, views, and other relations that hold data should have singular names, not plural.